sql/core/src/main/scala/org/apache/spark/sql/execution/joins/SortMergeJoinExec.scala

```diff
@@ -45,6 +45,27 @@ case class SortMergeJoinExec(
   override lazy val metrics = Map(
     "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"))
 
+  override def simpleString(maxFields: Int): String = {
```

Comment:
why override it? I think the new explain format won't call it?

Reply:
@cloud-fan Oh.. boy... many thanks for catching this. I will remove.